int32 id
string name
---
string info
